contributing/core_team.rst

```diff
@@ -178,6 +200,13 @@ A pull request **can be merged** if:
   ``+1`` (if the submitter is part of the Mergers team, two *other* members)
   and no Core member voted ``-1`` (via GitHub reviews or as comments).
 
+.. _core-team_minor-changes:
+
+.. note::
+
+    Minor changes comprise typos, DocBlock fixes, code standards
+    violations, and minor CSS, JavaScript and HTML modifications.
```

Comment:
This directive is moved up from the bottom of the document.
It's a bit unfortunate that we also call this 'minor changes' (conflicting the definition in `gh` documented below), but I can't think of a better name.

Reply:
"supporting changes" ? 
"Peripheral changes" ?